ubuntu-headless-qt/README.md

```diff
@@ -0,0 +1,996 @@
+ - MIPI DSI enabled
+ - Bootloader with U-Boot Fastboot UDP enabled.
+
+Known issues:
+
+ - Only support for 48 Khz audio sampling rate family.
+
+## Building Ubuntu
+**Step 1**: Linux Ubuntu 22.04 is recommended for the build. Prepare environment for building package and local build environment.
+Before starting the build, prepare a file named `core-image-qt-rzpi.tar.bz2`, which is the output from core-image-qt in yocto build and place in the same level of `main_script.sh`.
+
+User can modify the target wic/rootfs size, name, time zone and version of ubuntu base inside `config.ini`.
+
+Run the command below on the Linux Host PC to install packages to be used.
+```
+$ sudo apt-get update
+$ sudo apt-get install gawk wget git-core diffstat unzip texinfo gcc-multilib \
+build-essential chrpath socat cpio python python3 python3-pip python3-pexpect \
+xz-utils debianutils iputils-ping libsdl1.2-dev xterm p7zip-full
+```
+
+Run the below commands to set the user name and email address before starting the build procedure.
+```
+$ git config --global user.email "you@example.com"
+$ git config --global user.name "Your Name"
+```
+**Step 2**: Build package
+First, we need to change the permisstions of `main_script.sh` to make it executable.
+Then we can execute the script as follows:
+```
+chmod +x main_script.sh
+./main_script.sh
+```
+**Note:**
+Please note that this build requires internet access and will take several hours.
+
+**Step 3**: Collect the output
+
+After building Ubuntu, the output can be collected as `*.tar.zst` and `*.wic` (compression file is `*.wic.tar.gz)
+
+**Note:**
+Please note that file name is modified by users in `config.ini`.
+
+## Hierarchy
+```
+ubuntu-headless-qt/
+├── config
+│   └── network_interfaces.conf
+├── config.ini
+├── include
+│   ├── 00_prepare_env.sh
+│   ├── 01_prepare_ubuntu_base.sh
+│   ├── 02_prepare_rootfs_qt.sh
+│   ├── 03_prepare_conf.sh
+│   ├── 04_mount.sh
+│   └── 05_create_wic.sh
+├── main_script.sh
+├── README.md
+└── script
+    ├── apt_install_base.sh
+    └── set_root_password.sh
+
+3 directories, 12 files
+```
+
+**Output folder outline:**
+```
+ubuntu-headless-qt/
+|-- README.md                               <---- This file
+|-- config
+|   `-- network_interfaces.conf
+|-- config.ini                              <---- User configuration
+|-- core-image-qt-rzpi.tar.bz2              <---- Prepared input
+|-- include
+|   |-- 00_prepare_env.sh
+|   |-- 01_prepare_ubuntu_base.sh
+|   |-- 02_prepare_rootfs_qt.sh
+|   |-- 03_prepare_conf.sh
+|   |-- 04_mount.sh
+|   `-- 05_create_wic.sh
+|-- main_script.sh
+|-- qt_rootfs_source
+|-- rootfs
+|-- rootfs.tar.zst                          <---- Output rootfs
+|-- script
+|   |-- apt_install_base.sh
+|   `-- set_root_password.sh
+|-- ubuntu-base-22.04-base-arm64.tar.gz
+|-- ubuntu-image-qt-rzpi.wic                <---- Output WIC
+`-- ubuntu-image-qt-rzpi.wic.tar.gz         <---- Output compressed WIC
+```
+### U-boot environment
```

Comment:
How can I prepare U-Boot environment?

You can ask users refer to the original package as all images have same the procudure

Reply:
Please point out the link for "the original packages".

Reply:
Do you mean the link to Readme in QTv1.1.1 package ?

Reply:
I accept this change for now. I'm not sure how we organize the release package. Let's discuss with US team on this